corehq/apps/export/models/new.py

```diff
@@ -1372,9 +1371,17 @@ def get_query(self, include_filters=True):
         return query
 
 
+def datasource_export_instance_config_class():
+    """
+    Ugly, but this is to circumvent a circular import
+    """
+    from corehq.apps.userreports.models import DataSourceConfiguration
+    return DataSourceConfiguration
+
+
 class DataSourceExportInstance(ExportInstance):
     type = 'datasource'
-    config = SchemaProperty(DataSourceConfiguration)
+    config = SchemaProperty(datasource_export_instance_config_class)
```

Comment:
Does this work? Can you resolve the circular import by moving this class definition somewhere else?